.github/workflows/deploy-blog-in-stage.yaml

```diff
@@ -11,9 +11,7 @@ env:
 jobs:
   build-and-push-prod:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
+    permissions: write-all
```

Comment:
from : https://github.com/orgs/community/discussions/57724#discussioncomment-6225246
however, not a big fan tho...